spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/export/signalfx/SignalFxMetricsExportAutoConfigurationTests.java

```diff
@@ -50,7 +49,7 @@
 	@Test
 	public void backsOffWithoutAClock() {
 		this.runner.run((context) -> assertThat(context)
-				.doesNotHaveBean(InfluxMeterRegistry.class));
+				.doesNotHaveBean(SignalFxMeterRegistry.class));
```

Comment:
Nit on similar SignalFx test that was referring to the wrong type.

Reply:
Oops. Thanks for the catch.